admiral/pkg/client/loader/kube_loader.go

```diff
@@ -85,6 +86,19 @@ func (loader *KubeClientLoader) LoadKubeClientFromConfig(config *rest.Config) (k
 	return kubernetes.NewForConfig(config)
 }
 
+func (loader *KubeClientLoader) LoadNumaflowClientFromPath(kubeConfigPath string) (numaflow.Interface, error) {
```

Comment:
should we have this in a new file called - `numaflow_loader.go`?

Reply:
I see istio client loaded in the same file, do you see an advantage in separating it out?

Reply:
Got it, if it has istio client we can keep this here as well. I thought the file is only for k8s client.